data/card_catalogue/upgradeable-1.json

```diff
@@ -60,7 +60,7 @@
         "active": "Deal 3 damage to a visible character with a painful bolt of electricity",
         "passive": null
       },
-      "flavour": "A smile costs less than electricity, but gives much light",
```

Comment:
Included this by accident in this PR. This was just a shitpost originally, replacing it with something less ironic